packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -138,56 +148,65 @@ interface DesktopMapOverlaySelectorProps {
   entityName?: Entity['name'];
   overlayLibraryOpen: boolean;
   toggleOverlayLibrary: () => void;
+  setIsOpen: any;
 }
 
 export const DesktopMapOverlaySelector = ({
   overlayLibraryOpen,
   toggleOverlayLibrary,
-}: DesktopMapOverlaySelectorProps) => {
-  const { projectCode, entityCode } = useParams();
-  const { hasMapOverlays } = useMapOverlays(projectCode, entityCode);
-  const { data: mapOverlayData } = useMapOverlayReport();
-
-  return (
-    <Wrapper>
-      <Header>
-        <Heading>Map Overlays</Heading>
-      </Header>
-      <Container>
-        <TitleWrapper>
-          <MapOverlaySelectorTitle />
-          <MapOverlayDatePicker />
-        </TitleWrapper>
-        {hasMapOverlays && (
-          <OverlayLibraryAccordion
-            expanded={overlayLibraryOpen}
-            onChange={toggleOverlayLibrary}
-            square
-          >
-            <OverlayLibraryHeader
-              expandIcon={<ArrowDropDown />}
-              aria-controls="overlay-library-content"
-              id="overlay-library-header"
+  setIsOpen,
+}: DesktopMapOverlaySelectorProps) =>
+  // { setIsOpen }: any,
+  {
+    const { projectCode, entityCode } = useParams();
+    const { hasMapOverlays } = useMapOverlays(projectCode, entityCode);
+    const { data: mapOverlayData } = useMapOverlayReport();
+    const handleOpen = () => {
+      setIsOpen(true);
+    };
+    return (
+      <Wrapper>
+        <Header>
+          <Heading>Map Overlays</Heading>
+          <Tooltip arrow interactive placement="top" title="Generate Report">
+            <TableAssignmentIcon onClick={() => handleOpen()} />
```

Comment:
Since this is acting like a button, this should probably be wrapped in a `button` element. In this case, then you can set `title` on the `button` to be `Generate report` and not need to create a `Tooltip`